src/bots/ping/tests/handler.test.js

```diff
@@ -0,0 +1,11 @@
+import client from '../../../client';
+import handler from '../handler';
+
+describe('ping event', () => {
+  test('ping handler is called when ping event is emitted', () => {
+    const logSpy = jest.spyOn(client.logger, 'debug');
```

Comment:
Can you not see the logs in the test themselves instead of spying?

Reply:
The log function here is mostly just used as _a function_ to spy on, not for its actual logging properties. I've considered changing it to a variadic function that does nothing, but that seems like a hack. 

We should look at the Jest docs for a better way to do this than spying on a logging function.

Reply:
Nope

Reply:
Oh I see what you meant Kevin. And yes there's a better way that I forgot to mention - use jest.fn()